.github/workflows/autotest_prs.yml

```diff
@@ -45,6 +45,14 @@ jobs:
         go install honnef.co/go/tools/cmd/staticcheck@latest
         staticcheck ./...
 
+    - name: Setup MinIO
```

Comment:
If you feel like yielding to an inner grammarian, you could use "Set up" as the verb (versus "Setup" the noun). :-)

Reply:
up dated!